config/initializers/content_security_policy.rb

```diff
@@ -12,6 +12,9 @@
   policy.script_src  :self, :https
   policy.style_src   :self, :https
 
+  # Enable trusted types enforcement in browsers that support it
+  policy.require_trusted_types_for "'script'"
```

Comment:
This is not yet available in rails. It was [added to master](https://github.com/rails/rails/commit/0f7931572bb5273f71f26a4e92c34da599ec186b#diff-4933931ba84ddfd7a371089890f6dbd673447a8a52af8eb8fd85136d3fdec053), which means it will be out when rails 7 launches.

Reply:
@tsmartt ah thanks, let me rework this then.